stable/elasticsearch-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: elasticsearch-exporter
-version: 1.1.3
+version: 1.1.4
```

Comment:
Bump a minor version instead of patch version since this is a new feature.

Reply:
```suggestion
version: 1.1.4
```
Bumped up to 1.2.0